site/dat/repo/various.json

```diff
@@ -2099,5 +2099,29 @@
         "downloadUrl": "https://github.com/Qytera-Gmbh/JMeterHARImporterPlugin/releases/download/v0.2.3/jmeter-har-import-plugin-v0.2.3.jar"
       }
     }
+  },
+  {
+    "id": "JmeterNamesTemplater",
+    "name": "JMeter Names Templater",
+    "description": "Jmeter Names Templater is a plugin for Jmeter that allows you to set naming templates for elements and rename them based on a described configuration file.",
+    "vendor": "Daniil Golikov",
+    "markerClass": "com.example.jmeter.plugin.gui.RenameElementsMenuCreator",
+    "helpUrl": "https://github.com/DanilGolikov/JmeterNamesTemplater/blob/master/README.md",
```

Comment:
`https://github.com/DanilGolikov/Custom_jmeter_functions` should be simpler and sufficient URL here.

Also I'd work on that page for better presentation, titles and help document structure. Up to you, though.